plugin/src/main/java/io/jenkins/plugins/casc/Configurator.java

```diff
@@ -156,6 +157,15 @@ default Class getImplementedAPI() {
      * Only export attributes which are <b>not</b> set to default value.
      */
     @CheckForNull
-    CNode describe(T instance, ConfigurationContext context) throws Exception;
+    default CNode describe(T instance, ConfigurationContext context) throws Exception {
```

Comment:
@ndeloof do you have anything against a default describe? This can be used for basic cases.

`MavenConfigurator` would need a custom but `Jenkins`, `AdminWhitelistRuleConfigurator`, `UpdateCenterConfigurator`, `UpdateSiteConfigurator` seems to settle for default case